package.json

```diff
@@ -37,20 +37,27 @@
     "@types/jest": "^27.4.0",
     "@typescript-eslint/eslint-plugin": "^5.10.0",
     "@typescript-eslint/parser": "^5.10.0",
+    "axios": "^0.26.0",
```

Comment:
Just a little rundown on `devDependencies` vs. `dependencies`. `devDependencies` are packages that are only used for development/building. `dependencies` will be packages that will get built into the library and are needed for the library to successfully function. So `axios` will most likely have to go into `dependencies` since it will be needed to be packaged into the SDK for it to work correctly.